File: app/src/main/java/net/devdome/paperplayer/playback/events/PlayAllSongs.java
package net.devdome.paperplayer.playback.events;

import android.util.Log;

/**
 * PaperPlayer Michael Obi 14 01 2017 10:33 AM
 */

public class PlayAllSongs {
    private static final String TAG = "PlayAllSongs";
    /**
     * ID of SOng to start play with
     */
    private long startSongId;

    public PlayAllSongs(long startSongId) {
        this.startSongId = startSongId;
    }

    public long getStartSongId() {
        Log.d(TAG, "getStartSongId() called");
        return this.startSongId;
    }
}

File: app/src/main/java/net/devdome/paperplayer/playback/events/action/PlayAllSongs.kt
package net.devdome.paperplayer.playback.events.action

/**
 * PaperPlayer Michael Obi 14 01 2017 10:33 AM
 */

class PlayAllSongs(private val startSongId: Long) {

    fun getStartSongId(): Long {
        return this.startSongId
    }
}
